src/TMS/TMS.cpp

```diff
@@ -19,6 +19,8 @@ void TMS::updateTemps() {
             mux2.writePin(IO::GPIO::State::LOW);
         }
         thermTemps[i] = thermistor.getTempCelcius();
+
+        thermVoltages[i] = (thermistor.getRawADC()) * 3300 / 4095;
```

Comment:
Could you add a comment to this line explaining what the 3300 and 4095 are? Just incase someone comes back to this in the future